src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/PathFigure.cs

```diff
@@ -223,10 +223,7 @@ internal bool CanSerializeToString()
         internal string ConvertToString(string format, IFormatProvider provider)
         {
             PathSegmentCollection segments = Segments;
-            return "M" + 
-                ((IFormattable)StartPoint).ToString(format, provider) + 
-                (segments != null ? segments.ConvertToString(format, provider) : "") +
-                (IsClosed ? "z" : "");
+            return $"M{((IFormattable)StartPoint).ToString(format, provider)}{(segments != null ? segments.ConvertToString(format, provider) : "")}{(IsClosed ? "z" : "")}";
```

Comment:
Again, this seems hard to be understood. Maybe consider using string.Concat / string.Format here.

Reply:
Same here, we can just use Raw String Literals lang feature and combine it with interpolation, no need for string.Format ugliness.